src/component/project/ProjectAccess/ProjectAccessAssign/ProjectAccessAssign.tsx

```diff
@@ -109,8 +110,7 @@ export const ProjectAccessAssign = ({
     const { refetchProjectAccess } = useProjectAccess(projectId);
     const { addAccessToProject, changeUserRole, changeGroupRole, loading } =
         useProjectApi();
-    const { users = [] } = useUsers();
-    const { groups = [] } = useGroups();
+    const { users = [], groups = [] } = useAccess();
```

Comment:
Is this default needed? It seems to me that it's already setting a default in the useAccess hook.

Reply:
It is not, good catch.